tests/components/drop_connect/test_coordinator.py

```diff
@@ -1,74 +1,65 @@
 """Test DROP coordinator."""
-from homeassistant.components.drop_connect.const import DOMAIN
 from homeassistant.config_entries import ConfigEntryState
 from homeassistant.const import STATE_UNAVAILABLE, STATE_UNKNOWN
 from homeassistant.core import HomeAssistant
-from homeassistant.setup import async_setup_component
 
-from .common import TEST_DATA_HUB, TEST_DATA_HUB_RESET, TEST_DATA_HUB_TOPIC
+from .common import (
+    TEST_DATA_HUB,
+    TEST_DATA_HUB_RESET,
+    TEST_DATA_HUB_TOPIC,
+    config_entry_hub,
+)
 
 from tests.common import async_fire_mqtt_message
 from tests.typing import MqttMockHAClient
 
 
-async def test_bad_json(
-    hass: HomeAssistant, config_entry_hub, mqtt_mock: MqttMockHAClient
-) -> None:
+async def test_bad_json(hass: HomeAssistant, mqtt_mock: MqttMockHAClient) -> None:
```

Comment:
I think we can put this test in every platform as every platform has their own way to handle it probably